src/vm/methodtablebuilder.h

```diff
@@ -2956,6 +2956,11 @@ class MethodTableBuilder
                                 , AllocMemTracker *pamTracker
         );
 
+#if defined(_TARGET_X86_) || defined(_TARGET_AMD64_) 
+private:
+    bool isX86IntrinsicType;
```

Comment:
This should be called `isHardwareIntrinsic` and it should not be under ifdef - we will want the same bool for other platform intrinsics, and it should be in say bmtProperties to follow the style of this type.

Reply:
> it should be in say bmtProperties to follow the style of this type.

Don't you mean that `isHardwareIntrinsic` should be a field of `bmtProperties` struct?

Reply:
Right - it is what I meant

Reply:
Thanks, will do.